### PR TITLE
TSERV-185 lower case names for REST methods

### PR DESCRIPTION
**Pull request by prakashmrec**

### CODE REVIEW

File: test-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/RestRequestDslTest.groovy [hunk 1]
```diff
@@ -59,35 +60,81 @@ class RestRequestDslTest {
...
+        assert restRequest.followRedirects: 'Not respecting followRedirects'
+        assert restRequest.entitizeParameters: 'Not respecting entitizeParameters'
+        assert restRequest.postQueryString: 'Not respecting postQueryString'
         assert restRequest.timeout == '5000'
     }
 
     @Test
+    void addsHeaders() throws Exception {
+        TestRecipe recipe = recipe {
+            //Bug in the IntelliJ Groovyc - need parentheses here to make it compile!
```
Comment (spindelmanne) [inline test-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/RestRequestDslTest.groovy]: We should remove this comment.

File: test-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/RestRequestDslTest.groovy [hunk 2]
```diff
@@ -59,35 +60,81 @@ class RestRequestDslTest {
...
+        RestTestRequestStep restRequest = extractFirstTestStep(recipe) as RestTestRequestStep
+        assert restRequest.headers['Cache-Control'] == ['nocache']
+        assert restRequest.headers['Expires'] == ['0']
+        assert restRequest.headers['Accepts'] == ['text/plain', 'text/xml']
+    }
+
+    @Test
+    void createsParameters() throws Exception {
+        TestRecipe recipe = recipe {
+            //Bug in the IntelliJ Groovyc - need parentheses here to make it compile!
```
Comment (spindelmanne) [inline test-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/RestRequestDslTest.groovy]: Here too

File: test-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/RestRequestDslTest.groovy [hunk 3]
```diff
@@ -59,35 +60,81 @@ class RestRequestDslTest {
...
+        assert parameters[2].value == 'abc'
+        assert parameters[3].type == RestParameter.TypeEnum.HEADER
+        assert parameters[3].name == 'X-My-Header'
+        assert parameters[3].value == 'headervalue'
+    }
+
+    @Test
     void createsStatusAssertions() throws Exception {
         TestRecipe recipe = recipe {
             //Bug in the IntelliJ Groovyc - need parentheses here to make it compile!
```
Comment (spindelmanne) [inline test-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/RestRequestDslTest.groovy]: Here too (just search for it)

Comment (prakashmrec) [inline test-dsl/src/test/groovy/com/smartbear/readyapi4j/dsl/RestRequestDslTest.groovy]: Fixed.

**Comment by spindelmanne** [comment]
Looks good now!

**Pull request merged**